CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+- Add support for the address ownership proof protocol (AOPP), i.e.: handle 'aopp:?...' URIs in the BitBoxApp.
```

Comment:
Add a link to https://aopp.group?
Also, normally spelled upper case Address Ownership ...

Reply:
also, " ... in the BitBoxApp" - this **is** the app changelog :)